core/app/[locale]/(default)/account/(tabs)/addresses/page.tsx

```diff
@@ -24,13 +26,17 @@ export async function generateMetadata() {
   };
 }
 
+export enum AddressPageLimit {
+  Default = 3,
```

Comment:
Just curious if you discuss the qty with Turner or Eric? I believe previously we have increased it to 10🤔

Reply:
You are right, I changed it to 3 during work and local testing. I changed it back to 10, thanks for noticing.